### ISSUE TITLE
README is missing package needed to install pdd

### ISSUE THREAD

**Issue by longtimeago** [teamed/pdd#47]
README is missing one package needed to install pdd - `zlib1g-dev`

**Comment by davvd** [comment]
@yegor256 please do something about this issue

**Comment by yegor256** [comment]
@longtimeago I don't think it's needed... check `.travis.yml`. it's not used there and the build is clean. are you sure you can't install it?

**Comment by longtimeago** [comment]
@yegor256 I've reinstalled my system from scratch recently. Without that lib I was unable to compile native extensions for PDD or some it's dependency

**Comment by yegor256** [comment]
@longtimeago how exactly do you compile it? `bundle update`? I mean, how can I reproduce? all I have is `Travis` and it works fine. Rultor too, with a clean Ubuntu 14

**Comment by longtimeago** [comment]
@yegor256 I just run ```gem install pdd```

**Comment by yegor256** [comment]
@longtimeago OK, let's try this: 668088c

**Comment by yegor256** [comment]
@rultor release, tag is `0.15.3`

**Comment by yegor256** [comment]
@rultor release, tag is `0.15.3`

**Comment by yegor256** [comment]
@rultor release, tag is `0.15.4`

**Comment by yegor256** [comment]
@longtimeago see? `gem install pdd` works just fine.

**Comment by longtimeago** [comment]
@yegor256 I suspect that `zlib1g-dev` is already on `yegor256/rultor` docker image

**Comment by longtimeago** [comment]
@yegor256 here is the log from fresh Ubuntu 14.04.03 installed on VB
```
ubuntu@ubuntu:~$ sudo apt-get install -y libmagic1 libmagic-dev ruby-dev
Reading package lists... Done
Building dependency tree      
Reading state information... Done
libmagic1 is already the newest version.
The following extra packages will be installed:
  libruby1.9.1 libyaml-0-2 ruby ruby1.9.1 ruby1.9.1-dev
Suggested packages:
  ri ruby1.9.1-examples ri1.9.1 graphviz ruby-switch
The following NEW packages will be installed:
  libmagic-dev libruby1.9.1 libyaml-0-2 ruby ruby-dev ruby1.9.1 ruby1.9.1-dev
0 upgraded, 7 newly installed, 0 to remove and 256 not upgraded.
Need to get 3,673 kB of archives.
After this operation, 17.4 MB of additional disk space will be used.
Get:1 http://security.ubuntu.com/ubuntu/ trusty-security/main libyaml-0-2 amd64 0.1.4-3ubuntu3.1 [48.1 kB]
Get:2 http://security.ubuntu.com/ubuntu/ trusty-security/main libmagic-dev amd64 1:5.14-2ubuntu3.3 [62.8 kB]
Get:3 http://security.ubuntu.com/ubuntu/ trusty-security/main ruby1.9.1 amd64 1.9.3.484-2ubuntu1.2 [35.6 kB]
Get:4 http://security.ubuntu.com/ubuntu/ trusty-security/main libruby1.9.1 amd64 1.9.3.484-2ubuntu1.2 [2,645 kB]
Get:5 http://security.ubuntu.com/ubuntu/ trusty-security/main ruby1.9.1-dev amd64 1.9.3.484-2ubuntu1.2 [871 kB]
Get:6 http://archive.ubuntu.com/ubuntu/ trusty/main ruby all 1:1.9.3.4 [5,334 B]
Get:7 http://archive.ubuntu.com/ubuntu/ trusty/main ruby-dev all 1:1.9.3.4 [4,660 B]
Fetched 3,673 kB in 1min 58s (31.0 kB/s)
Selecting previously unselected package libyaml-0-2:amd64.
(Reading database ... 170586 files and directories currently installed.)
Preparing to unpack .../libyaml-0-2_0.1.4-3ubuntu3.1_amd64.deb ...
Unpacking libyaml-0-2:amd64 (0.1.4-3ubuntu3.1) ...
Selecting previously unselected package libmagic-dev:amd64.
Preparing to unpack .../libmagic-dev_1%3a5.14-2ubuntu3.3_amd64.deb ...
Unpacking libmagic-dev:amd64 (1:5.14-2ubuntu3.3) ...
Selecting previously unselected package ruby.
Preparing to unpack .../ruby_1%3a1.9.3.4_all.deb ...
Unpacking ruby (1:1.9.3.4) ...
Selecting previously unselected package ruby1.9.1.
Preparing to unpack .../ruby1.9.1_1.9.3.484-2ubuntu1.2_amd64.deb ...
Unpacking ruby1.9.1 (1.9.3.484-2ubuntu1.2) ...
Selecting previously unselected package libruby1.9.1.
Preparing to unpack .../libruby1.9.1_1.9.3.484-2ubuntu1.2_amd64.deb ...
Unpacking libruby1.9.1 (1.9.3.484-2ubuntu1.2) ...
Selecting previously unselected package ruby1.9.1-dev.
Preparing to unpack .../ruby1.9.1-dev_1.9.3.484-2ubuntu1.2_amd64.deb ...
Unpacking ruby1.9.1-dev (1.9.3.484-2ubuntu1.2) ...
Selecting previously unselected package ruby-dev.
Preparing to unpack .../ruby-dev_1%3a1.9.3.4_all.deb ...
Unpacking ruby-dev (1:1.9.3.4) ...
Processing triggers for man-db (2.6.7.1-1ubuntu1) ...
Setting up libyaml-0-2:amd64 (0.1.4-3ubuntu3.1) ...
Setting up libmagic-dev:amd64 (1:5.14-2ubuntu3.3) ...
Setting up ruby (1:1.9.3.4) ...
Setting up libruby1.9.1 (1.9.3.484-2ubuntu1.2) ...
Setting up ruby1.9.1-dev (1.9.3.484-2ubuntu1.2) ...
Setting up ruby-dev (1:1.9.3.4) ...
Setting up ruby1.9.1 (1.9.3.484-2ubuntu1.2) ...
Processing triggers for libc-bin (2.19-0ubuntu6.6) ...
ubuntu@ubuntu:~$ sudo gem install pdd
Fetching: mini_portile2-2.0.0.rc2.gem (100%)
Fetching: nokogiri-1.6.7.1.gem (100%)
Building native extensions.  This could take a while...
ERROR:  Error installing pdd:
    ERROR: Failed to build gem native extension.

        /usr/bin/ruby1.9.1 extconf.rb
checking if the C compiler accepts ... yes
Building nokogiri using packaged libraries.
Using mini_portile version 2.0.0.rc2
checking for gzdopen() in -lz... no
zlib is missing; necessary for building libxml2
*** extconf.rb failed ***
Could not create Makefile due to some reason, probably lack of
necessary libraries and/or headers.  Check the mkmf.log file for more
details.  You may need configuration options.

Provided configuration options:
    --with-opt-dir
    --without-opt-dir
    --with-opt-include
    --without-opt-include=${opt-dir}/include
    --with-opt-lib
    --without-opt-lib=${opt-dir}/lib
    --with-make-prog
    --without-make-prog
    --srcdir=.
    --curdir
    --ruby=/usr/bin/ruby1.9.1
    --help
    --clean
    --use-system-libraries
    --enable-static
    --disable-static
    --with-zlib-dir
    --without-zlib-dir
    --with-zlib-include
    --without-zlib-include=${zlib-dir}/include
    --with-zlib-lib
    --without-zlib-lib=${zlib-dir}/lib
    --enable-cross-build
    --disable-cross-build


Gem files will remain installed in /var/lib/gems/1.9.1/gems/nokogiri-1.6.7.1 for inspection.
Results logged to /var/lib/gems/1.9.1/gems/nokogiri-1.6.7.1/ext/nokogiri/gem_make.out
```
After that I've installed ``` zlib1g-dev ``` and only then command ``` gem install pdd ``` was succeed

**Comment by yegor256** [comment]
@longtimeago you're right: https://github.com/yegor256/rultor/blob/master/src/docker/Dockerfile#L66

**Comment by yegor256** [comment]
@longtimeago fixed in aa3d068 thanks
